scripts/.env.instantiate.example

```diff
@@ -0,0 +1,26 @@
+NEUTRON_RPC="https://rpc-falcron.pion-1.ntrn.tech:443"
+GAS_PRICES=0.02
+NEUTRON_CHAIN_ID="pion-1"
+TARGET_CHAIN_ID="test-2"
+KEYRING_BACKEND="test"
+DEPLOY_WALLET="drop-deploy"
+MIN_NTRN_REQUIRED=10
+TARGET_SDK_VERSION="0.47.10"
+TARGET_BASE_DENOM="uatom"
+NEUTRON_SIDE_TRANSFER_CHANNEL_ID="channel-788"
+IBC_REGISTER_FEE="1000000"
```

Comment:
I don't like how some of these settings are duplicated in both env examples. Maybe we need a third, shared one?

Reply:
Not sure, it may lead to confusions with configuration. Also potentially these two configurations may be stand alone.